askama/Cargo.toml

```diff
@@ -29,6 +29,12 @@ with-axum = ["askama_derive/with-axum"]
 with-rocket = ["askama_derive/with-rocket"]
 with-warp = ["askama_derive/with-warp"]
 
+## Enables the ability to put templates in a directory relative to the source file that uses them.
+## Requires a nightly compiler and adding:
+## RUSTFLAGS='--cfg proc_macro_span --cfg procmacro2_semver_exempt'
+## to your cargo build command.
```

Comment:
Consider adding the `document-features` crate so this info can automatically populate the lib.rs doc comment without having to duplicate it.